week4/이지영/H-Index.js

```diff
@@ -0,0 +1,10 @@
+function solution(citations) {
+  let answer = 0;
+  citations.sort((a, b) => b - a);
```

Comment:
nitpick: 이 예시는 상관없다만, 불변성을 고려하는 코드를 생각해보는것도 괜찮을것같아요! 
```suggestion
const sorted = [...citations].sort((a, b) => b - a);
```